oathkeeper/access-rules/kavach-web.json

```diff
@@ -32,7 +32,7 @@
       "url": "http://kavach-web:3000"
     },
     "match": {
-      "url": "http://127.0.0.1:4455/.factly/kavach/web<(/(organisation|organisation/edit|organisation(/(\\d+/settings(/(tokens|tokens/create|roles|roles(/(\\d+/users|\\d+/create|\\d+/edit))|policies|users|users/new))))|settings|password|profile|profile/invite|users|users(/new)|applications(/(create|type|\\d+/edit|\\d+/detail|\\d+/settings|\\d+/settings(/(users|tokens|tokens/create|roles|roles/create|policies|policies/create|spaces|spaces(/(create|\\d+/settings|\\d+/settings(/(tokens|roles|policies|users))))))))?))?/?>",
+      "url": "http://127.0.0.1:4455/.factly/kavach/web<(/(organisation|organisation/edit|organisation/create|organisation(/(\\d+/settings(/(tokens|tokens/create|roles|roles/create|roles(/(\\d+/users|\\d+/create|\\d+/edit))|policies|policies/create|policies(/(\\d+/view|\\d+/edit))|users|users/new))))|settings|password|profile|profile/invite|users|users(/new)|applications(/(create|type|\\d+/edit|\\d+/detail|\\d+/settings|\\d+/settings(/(users|tokens|tokens/create|roles|roles/create|roles(/(\\d+/users|\\d+/create|\\d+/edit))|policies|policies/create|policies(/(\\d+/view|\\d+/edit))|spaces|spaces(/(create|\\d+/edit|\\d+/settings|\\d+/settings(/(tokens|tokens/create|roles|roles/create|roles(/(\\d+/users|\\d+/create|\\d+/edit))|policies|policies/create|policies(/(\\d+/view|\\d+/edit))|users))))))))?))?/?>",
```

Comment:
@ishpaul777 regex can be improvised like `organisation|organisation/edit|organisation/create`
to organisation(/edit|/create)(/\)

roles(/(\\d+/users|\\d+/create|\\d+/edit) to  roles(/(\\d+/(/users|/create|/edit))